router/types/types.go

```diff
@@ -33,6 +33,10 @@ type Route struct {
 	// Sticky is whether or not to use sticky sessions for this route. It is only
 	// used for HTTP routes.
 	Sticky bool `json:"sticky,omitempty"`
+	// Path is the optional prefix to route to this service. It's exclusive with
+	// the TLS options and can only be set if a "default" route with the same domain
+	// and no Path already exists in the route table.
```

Comment:
Probably worth mentioning it must have both a leading and trailing slash (is that necessary?).